cranelift/codegen/src/isa/x64/inst/emit.rs

```diff
@@ -1527,6 +1527,7 @@ pub(crate) fn emit(
                 SseOpcode::Cvtdq2pd => (LegacyPrefixes::_F3, 0x0FE6, 2),
                 SseOpcode::Cvtpd2ps => (LegacyPrefixes::_66, 0x0F5A, 2),
                 SseOpcode::Cvtps2pd => (LegacyPrefixes::None, 0x0F5A, 2),
+                SseOpcode::Cvtdq2ps => (LegacyPrefixes::None, 0x0F5B, 2),
```

Comment:
In addition to adding it here, can we remove it from [here](https://github.com/bytecodealliance/wasmtime/blob/0b1f51f80427a79fbd7d77849f7e2301a1d128f5/cranelift/codegen/src/isa/x64/inst/emit.rs#L1628) and make sure we switch over any uses of that (tweaking any handwritten uses of it in-place is fine for now)? Since the opcode has unary (one src, one dst) semantics it should only appear here, not elsewhere.

I'm also slightly suspect of the two `cvt*` opcodes above that one in `XmmRmR` too, and we should see if they should be moved over too, while we're at it...